fhir-codegen/gradle-plugin/src/main/kotlin/com/google/fhir/codegen/ModelTypeSpecGenerator.kt

```diff
@@ -84,6 +84,12 @@ class ModelTypeSpecGenerator(
             // with '_'.
             if (structureDefinition.name == "Element") {
               addModifiers(KModifier.OPEN)
+              // Override equals and hashcode methods for open classes to allow for properties
```

Comment:
here you're only doing this for element right? maybe refine the comment a bit because it makes it sounds like you're doing this for different classes.

also add the comment for line 104-107 where this is more accurate.

Reply:
Noted.